app/models/spree/app_configuration.rb

```diff
@@ -42,7 +42,7 @@ class AppConfiguration < Preferences::Configuration
     preference :checkout_zone, :string, default: nil
     preference :currency, :string, default: "USD"
     preference :currency_decimal_mark, :string, default: "."
-    preference :currency_symbol_position, :string, default: "before"
+    preference :currency_symbol_position, :string, default: "%u%n"
```

Comment:
I'll also convert from `before` to `%u%n` from `Spree::Money` instead. This way there's no chance of breaking any instance's configuration or having to migrate this setting to the new format.